packages/happo-target-firefox/src/waitForImagesToRender.js

```diff
@@ -1,11 +1,18 @@
 import findBackgroundImageUrls from './findBackgroundImageUrls';
 import getUrlsFromSrcset from './getUrlsFromSrcset';
 
-export function waitForImageToLoad(url) {
+export function waitForImageToLoad(imageOrUrl) {
```

Comment:
I'm not quite sure I follow why this needs to be polymorphic.

Reply:
It's called from a few more places (urls scraped from `srcset` and `background-image`). 

That being said, I'm not 100% happy with how this looks. Also, it makes me realize I probably need to do this for `srcset` as well.